src/components/common/ModalDialog/index.tsx

```diff
@@ -70,15 +53,21 @@ const ModalDialog = ({
   const fullScreen = useMediaQuery(theme.breakpoints.down('sm'))
 
   return (
-    <StyledDialog {...restProps} fullScreen={fullScreen} scroll="body" onClick={(e) => e.stopPropagation()}>
+    <Dialog
+      {...restProps}
+      fullScreen={fullScreen}
+      scroll={fullScreen ? 'paper' : 'body'}
```

Comment:
What's the reason behind switching the scroll container?

Reply:
It was the way I've found to solve a faulty behavior in responsiveness. I've considered it because we can scroll the whole paper if we are in fullScreen. I think the `scroll="body"` property aims to target scrolling solely in _> 600px_ resolutions.